robot_smach_states/src/robot_smach_states/world_model.py

```diff
@@ -27,34 +28,42 @@ def look_at_segmentation_area(robot, entity, volume=None):
     # Determine the height of the head target
     # Start with a default
     pos = entity.pose.frame.p
-    look_at_point_z = 0.7
+    x = pos.x()
+    y = pos.y()
+    z = 0.7
 
     # Check if we have areas: use these
     if volume in entity.volumes:
         search_volume = entity.volumes[volume]
         try:
-            look_at_point_z = search_volume.min_corner.z()
+            x_obj = 0.5 * (search_volume.min_corner.x() + search_volume.max_corner.x())
+            y_obj = 0.5 * (search_volume.min_corner.y() + search_volume.max_corner.y())
+            z_obj = search_volume.min_corner.z()
```

Comment:
Why not the average z? or the max? (i.e. top of the table)

Reply:
It's the bottom of the _volume_, not the _entity_.